<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="pathsLayout" rowstretch="0,0">
     <item row="0" column="1">
      <widget class="QToolButton" name="inputPathButton">
       <property name="toolTip">
        <string>Ctrl+I</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+I</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="inputPathEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Select .avs presets directory...</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="inputLabel">
       <property name="text">
        <string>Input</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="outputPathButton">
       <property name="toolTip">
        <string>Ctrl+O</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+O</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3" rowspan="2">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Settings...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="outputPathEdit">
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Same as input directory, or select different one...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="outputLabel">
       <property name="text">
        <string>Output</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout" stretch="0,0,0,0,0">
     <item>
      <widget class="QPushButton" name="listFilesButton">
       <property name="toolTip">
        <string>Ctrl+L</string>
       </property>
       <property name="text">
        <string>List Input Files</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+L</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="convertButton">
       <property name="toolTip">
        <string>Ctrl+Return</string>
       </property>
       <property name="text">
        <string>Convert</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Return</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="toolTip">
        <string>Esc</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="toolTip">
        <string>Ctrl+C</string>
       </property>
       <property name="text">
        <string>Clear Log</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="outputLayout" stretch="0,0">
     <item>
      <layout class="QVBoxLayout" name="logLayout" stretch="3,1">
       <item>
        <widget class="QTextEdit" name="logField">
         <property name="autoFormatting">
          <set>QTextEdit::AutoNone</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="threadLog">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="outputSampleLayout">
       <item>
        <layout class="QHBoxLayout" name="outputInfoLayout" stretch="0,1">
         <item>
          <widget class="QLabel" name="samplePresetLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="font">
            <font>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>Preview:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="samplePresetSelectBox">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="editable">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string/>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
           <property name="maxVisibleItems">
            <number>60</number>
           </property>
           <property name="minimumContentsLength">
            <number>1</number>
           </property>
           <property name="duplicatesEnabled">
            <bool>true</bool>
           </property>
           <property name="frame">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="outputSample">
         <property name="undoRedoEnabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>inputPathEdit</tabstop>
  <tabstop>outputPathEdit</tabstop>
  <tabstop>convertButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>clearButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>logField</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>280</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
